geode-core/src/main/java/org/apache/geode/internal/statistics/StatisticsMonitor.java

```diff
@@ -29,23 +29,20 @@
 
   private final Object mutex = new Object();
 
-  private volatile List<StatisticsListener> listeners = Collections.<StatisticsListener>emptyList();
+  private volatile ConcurrentHashSet<StatisticsListener> listeners =
```

Comment:
Instead of `volatile` I think this should be final.  I don't see a write to this var after initialization.